### PR TITLE
Enhance application action

### PR DESCRIPTION
**Pull request by abhinandan13jan**
## Fixes 

https://issues.redhat.com/browse/HACBS-1168
Following doc -> https://docs.google.com/document/d/1MkPSzk-pwzXeKT2oWrYUu87XYNtWlGkzhfnWPzBuRuI/edit#heading=h.2dz13zkk5prj

## Description

Enhances Application actions
Screenshot 2022-11-22 at 7 05 31 PM

## Type of change

- [ ] Feature
- [x] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Browser conformance: 

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: src/hacbs/components/ApplicationDetails/HacbsApplicationDetails.tsx [hunk 1]
```diff
@@ -101,24 +122,56 @@ const HacbsApplicationDetails: React.FC<HacbsApplicationDetailsProps> = ({ appli
...
+          },
           {
             type: 'separator',
             key: 'separator',
             label: '',
           },
           {
             key: 'application-quickstart',
-            label: 'Getting started with an app',
+            label: 'Getting started with an application',
```
Comment (christianvogt) [inline src/hacbs/components/ApplicationDetails/HacbsApplicationDetails.tsx:137]: The title of the quick start should also be updated to align:
image

File: src/hacbs/components/ApplicationDetails/HacbsApplicationDetails.tsx [hunk 2]
```diff
@@ -101,24 +122,56 @@ const HacbsApplicationDetails: React.FC<HacbsApplicationDetailsProps> = ({ appli
...
-            key: 'learning-resources',
-            label: 'Learning resources',
-            href: '',
-            isDisabled: true,
+            key: 'setup-lifecycle',
+            label: 'Setup application lifecycle',
+            component: (
+              <Link to={`/app-studio/applications/${applicationName}?activeTab=commits`}>
+                Setup application lifecycle
+              </Link>
```
Comment (christianvogt) [inline src/hacbs/components/ApplicationDetails/HacbsApplicationDetails.tsx:149]: I don't understand the logic of navigating to the commits page with the title `Setup application lifecycle`

Comment (abhinandan13jan) [reply src/hacbs/components/ApplicationDetails/HacbsApplicationDetails.tsx:149]: I just kept it as a link as there is nothing defined for this action in the UX doc cc: @Ranelim Please suggest

Comment (christianvogt) [reply src/hacbs/components/ApplicationDetails/HacbsApplicationDetails.tsx]: Until we have a use for this action, I suggest we simply comment it out for now.

Comment (abhinandan13jan) [reply src/hacbs/components/ApplicationDetails/HacbsApplicationDetails.tsx]: removed

**Pull request merged**